### ISSUE TITLE
Notice: Undefined index: opcache.load_comments in /var/www/*/src/OpCacheGUI/OpCache/Configuration.php on line 82

### ISSUE THREAD

**Issue by Quix0r** [PeeHaa/OpCacheGUI#76]
The said error comes with following PHP version (as FPM):
`PHP 7.0.14-1~dotdeb+8.1`
Latest `master` is in use.

Edit: It looks like that it needs to be **save**_comments, not **load**_comments:
```
$ php7.0 -i|grep opcache
/etc/php/7.0/cli/conf.d/10-opcache.ini,
opcache.blacklist_filename => no value => no value
opcache.consistency_checks => 0 => 0
opcache.dups_fix => Off => Off
opcache.enable => On => On
opcache.enable_cli => On => On
opcache.enable_file_override => Off => Off
opcache.error_log => no value => no value
opcache.fast_shutdown => 0 => 0
opcache.file_cache => no value => no value
opcache.file_cache_consistency_checks => 1 => 1
opcache.file_cache_only => 0 => 0
opcache.file_update_protection => 2 => 2
opcache.force_restart_timeout => 180 => 180
opcache.huge_code_pages => Off => Off
opcache.inherited_hack => On => On
opcache.interned_strings_buffer => 4 => 4
opcache.lockfile_path => /tmp => /tmp
opcache.log_verbosity_level => 1 => 1
opcache.max_accelerated_files => 2000 => 2000
opcache.max_file_size => 0 => 0
opcache.max_wasted_percentage => 5 => 5
opcache.memory_consumption => 4096M => 4096M
opcache.optimization_level => 0x7FFFBFFF => 0x7FFFBFFF
opcache.preferred_memory_model => no value => no value
opcache.protect_memory => 0 => 0
opcache.restrict_api => no value => no value
opcache.revalidate_freq => 2 => 2
opcache.revalidate_path => On => On
opcache.save_comments => 0 => 0
opcache.use_cwd => On => On
opcache.validate_permission => Off => Off
opcache.validate_root => Off => Off
opcache.validate_timestamps => On => On
$
```

**Comment by HiggsBoson** [comment]
Yes, load_comments is now always enabled, and the parameter is deprecated

http://git.php.net/?p=php-src.git;a=commitdiff;h=0a21a0c752a06d25704e04c15464700f75164849

**Comment by Quix0r** [comment]
So maybe remove it from GUI or check with `isset()` ?

**Comment by Quix0r** [comment]
Maybe duplicate of https://github.com/PeeHaa/OpCacheGUI/issues/69 ?

**Comment by ekinhbayar** [comment]
Closing as #74 is merged and the rest of changes required for this were done and merged in #85 , thanks!
